src/lib.rs

```diff
@@ -350,6 +347,8 @@ fn write_arg_markdown(buffer: &mut String, arg: &clap::Arg) -> fmt::Result {
         None => arg.get_id().to_string().to_ascii_uppercase(),
     };
 
+    // Markdown list item
+    write!(buffer, "* ")?;
```

Comment:
**Question** Is there a behavior reason to move this write of the `*`? It doesn't seem to change any of the tested behavior, so I just wanted to check if there was something I might not be seeing.

If its a question of style, I think I'd prefer this stay at the top of the function, so it's clear any `write!` calls done within the function are part of a bulleted list.

Reply:
Followup: Just noticed in 162c7a174f921ccef9003cc2a4fc919019e98bc0 that this was actually simply leftover from other work before this PR was split out, so I hope you don't mind I pushed a quick change to the branch reverting this minor movement.

Thank you again for this work! 🙂